[2024-11-21 22:08:11] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:08:33] INFO [users:12] User root accessed /users/auth/logout/
[2024-11-21 22:08:33] INFO [django.server:213] "POST /users/auth/logout/ HTTP/1.1" 302 0
[2024-11-21 22:08:33] INFO [users:12] User Anonymous accessed /
[2024-11-21 22:08:33] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 22:09:14] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 22:09:14] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3703
[2024-11-21 22:09:25] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 22:09:27] WARNING [users.views:55] Failed login attempt with username: root
[2024-11-21 22:09:27] INFO [django.server:213] "POST /users/login/ HTTP/1.1" 200 4025
[2024-11-21 22:09:33] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 22:09:35] INFO [users.views:48] User root logged in.
[2024-11-21 22:09:35] INFO [django.server:213] "POST /users/login/ HTTP/1.1" 302 0
[2024-11-21 22:09:35] INFO [users:12] User root accessed /
[2024-11-21 22:09:35] INFO [django.server:213] "GET / HTTP/1.1" 200 3269
[2024-11-21 22:10:06] INFO [users:12] User root accessed /
[2024-11-21 22:10:06] INFO [django.server:213] "GET / HTTP/1.1" 200 3269
[2024-11-21 22:12:27] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 22:12:29] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:15:07] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 22:15:08] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:16:43] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 22:16:44] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:18:45] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 22:18:46] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:20:45] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\custom_session_backend.py changed, reloading.
[2024-11-21 22:20:46] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:21:09] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 22:21:11] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:21:25] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:24:00] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\custom_session_backend.py changed, reloading.
[2024-11-21 22:24:02] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 22:24:08] INFO [users:12] User Anonymous accessed /
[2024-11-21 22:24:08] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 22:24:12] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 22:24:12] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5540
[2024-11-21 22:24:23] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 22:24:23] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5540
[2024-11-21 22:25:11] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 22:25:11] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3703
[2024-11-21 22:25:18] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 22:25:18] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5540
[2024-11-21 23:01:01] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:01:05] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:01:05] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5540
[2024-11-21 23:08:24] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 23:08:26] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:15:11] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:15:45] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:15:45] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4208
[2024-11-21 23:15:47] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:15:47] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4208
[2024-11-21 23:15:57] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:15:57] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:15:58] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:15:58] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:15:58] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:15:58] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/js/theme.js HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
[2024-11-21 23:15:58] INFO [django.server:213] "GET /static/admin/css/login.css HTTP/1.1" 200 951
[2024-11-21 23:16:11] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:16:12] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:16:12] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:16:12] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:16:12] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:16:12] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:17:56] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:17:57] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:17:57] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:17:57] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:17:57] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:17:57] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:19:02] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:19:03] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:19:03] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:19:03] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:19:03] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:19:03] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:19:31] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:19:33] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:19:33] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:19:33] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:19:33] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:19:33] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:21:37] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\models.py changed, reloading.
[2024-11-21 23:21:39] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:22:19] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:22:47] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:22:48] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:22:48] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:22:49] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:22:49] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:22:49] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:23:16] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:23:17] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:23:17] INFO [users:12] User Anonymous accessed /admin/
[2024-11-21 23:23:17] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-21 23:23:17] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:23:17] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:29:18] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:29:29] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:29:51] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:31:26] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:31:26] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:31:26] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4208
[2024-11-21 23:31:36] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:31:36] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/css/base.css HTTP/1.1" 200 22092
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2804
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/css/login.css HTTP/1.1" 200 951
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/css/responsive.css HTTP/1.1" 200 17972
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/js/theme.js HTTP/1.1" 200 1653
[2024-11-21 23:31:37] INFO [django.server:213] "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
[2024-11-21 23:31:37] INFO [users:12] User Anonymous accessed /favicon.ico
[2024-11-21 23:31:37] WARNING [django.request:248] Not Found: /favicon.ico
[2024-11-21 23:31:37] WARNING [django.server:213] "GET /favicon.ico HTTP/1.1" 404 2291
[2024-11-21 23:32:08] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-21 23:32:10] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-21 23:32:10] INFO [users:12] User admin accessed /admin/
[2024-11-21 23:32:10] INFO [django.server:213] "GET /admin/ HTTP/1.1" 200 7008
[2024-11-21 23:32:10] INFO [django.server:213] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[2024-11-21 23:32:10] INFO [django.server:213] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[2024-11-21 23:32:10] INFO [django.server:213] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [users:12] User admin accessed /admin/django_db_logger/statuslog/
[2024-11-21 23:32:16] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/ HTTP/1.1" 200 16006
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/actions.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/core.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /static/admin/js/filters.js HTTP/1.1" 304 0
[2024-11-21 23:32:16] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:32:17] INFO [django.server:213] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
[2024-11-21 23:32:17] INFO [django.server:213] "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 304 0
[2024-11-21 23:32:27] INFO [users:12] User admin accessed /admin/django_db_logger/statuslog/
[2024-11-21 23:32:27] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/?p=3 HTTP/1.1" 200 16275
[2024-11-21 23:32:28] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:32:28] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:32:29] INFO [users:12] User admin accessed /admin/django_db_logger/statuslog/
[2024-11-21 23:32:29] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/?p=4 HTTP/1.1" 200 16524
[2024-11-21 23:32:29] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:32:29] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:32:30] INFO [users:12] User admin accessed /admin/django_db_logger/statuslog/
[2024-11-21 23:32:30] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/?p=5 HTTP/1.1" 200 14713
[2024-11-21 23:32:31] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:32:31] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:34:25] INFO [users:12] User admin accessed /admin/users/user/
[2024-11-21 23:34:25] INFO [django.server:213] "GET /admin/users/user/ HTTP/1.1" 200 11538
[2024-11-21 23:34:25] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:34:25] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:34:25] INFO [django.server:213] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
[2024-11-21 23:34:25] INFO [django.server:213] "GET /static/admin/img/search.svg HTTP/1.1" 304 0
[2024-11-21 23:34:25] INFO [django.server:213] "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 304 0
[2024-11-21 23:34:55] INFO [users:12] User admin accessed /admin/auth/group/
[2024-11-21 23:34:55] INFO [django.server:213] "GET /admin/auth/group/ HTTP/1.1" 200 8443
[2024-11-21 23:34:55] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:34:55] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:34:58] INFO [users:12] User admin accessed /admin/django_db_logger/statuslog/
[2024-11-21 23:34:58] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/ HTTP/1.1" 200 16138
[2024-11-21 23:34:58] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:34:58] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:34:59] INFO [users:12] User admin accessed /admin/auth/
[2024-11-21 23:34:59] INFO [django.server:213] "GET /admin/auth/ HTTP/1.1" 200 5311
[2024-11-21 23:35:03] INFO [users:12] User admin accessed /admin/auth/group/add/
[2024-11-21 23:35:03] INFO [django.server:213] "GET /admin/auth/group/add/ HTTP/1.1" 200 12293
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/css/forms.css HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/js/SelectBox.js HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 304 0
[2024-11-21 23:35:03] INFO [django.server:213] "GET /static/admin/img/selector-icons.svg HTTP/1.1" 304 0
[2024-11-21 23:35:11] INFO [users:12] User admin accessed /admin/users/user/
[2024-11-21 23:35:11] INFO [django.server:213] "GET /admin/users/user/ HTTP/1.1" 200 11538
[2024-11-21 23:35:11] INFO [users:12] User admin accessed /admin/jsi18n/
[2024-11-21 23:35:11] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-21 23:36:40] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:36:40] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4966
[2024-11-21 23:36:41] INFO [django.server:213] "GET /static/images/favicon.png HTTP/1.1" 200 470
[2024-11-21 23:36:45] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:36:45] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4966
[2024-11-21 23:36:50] INFO [users:12] User admin accessed /users/login/
[2024-11-21 23:36:50] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 4461
[2024-11-21 23:36:55] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:36:55] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4966
[2024-11-21 23:44:59] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 23:45:00] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:46:33] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 23:46:35] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:50:28] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\views.py changed, reloading.
[2024-11-21 23:50:29] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:52:18] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\forms.py changed, reloading.
[2024-11-21 23:52:20] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:52:20] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:21] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:23] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:23] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:25] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:25] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:26] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:26] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:27] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:27] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:29] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:29] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:30] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:30] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:30] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:30] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:31] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:31] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:32] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:32] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:32] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:32] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:33] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:33] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:34] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:34] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:34] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:34] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:35] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:35] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:36] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:36] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:37] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:52:37] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:52:45] INFO [users:12] User admin accessed /
[2024-11-21 23:52:45] INFO [django.server:213] "GET / HTTP/1.1" 200 3270
[2024-11-21 23:52:47] INFO [users:12] User admin accessed /
[2024-11-21 23:52:47] INFO [django.server:213] "GET / HTTP/1.1" 200 3270
[2024-11-21 23:53:11] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:11] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:13] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:13] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:17] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:17] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:18] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\forms.py changed, reloading.
[2024-11-21 23:53:20] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:53:20] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:20] INFO [django.server:81] - Broken pipe from ('127.0.0.1', 55585)
[2024-11-21 23:53:21] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:21] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:22] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:22] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:24] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:24] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:25] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:25] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:53] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:53] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:53:54] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\views.py changed, reloading.
[2024-11-21 23:53:56] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:53:56] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:53:56] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:54:00] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:54:03] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:54:03] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:54:05] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:54:05] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:54:07] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:54:07] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:54:17] INFO [users:12] User admin accessed /users/register/
[2024-11-21 23:54:17] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5092
[2024-11-21 23:54:25] INFO [users:12] User admin accessed /
[2024-11-21 23:54:26] INFO [django.server:213] "GET / HTTP/1.1" 200 3270
[2024-11-21 23:54:30] INFO [users:12] User admin accessed /
[2024-11-21 23:54:30] INFO [django.server:213] "GET / HTTP/1.1" 200 3270
[2024-11-21 23:54:32] INFO [users:12] User admin accessed /users/profile/
[2024-11-21 23:54:32] INFO [users.views:115] User admin accessed their profile.
[2024-11-21 23:54:32] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2953
[2024-11-21 23:54:33] INFO [users:12] User admin accessed /users/auth/logout/
[2024-11-21 23:54:33] INFO [django.server:213] "POST /users/auth/logout/ HTTP/1.1" 302 0
[2024-11-21 23:54:33] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:54:33] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:54:57] INFO [users:12] User Anonymous accessed /users/profile/
[2024-11-21 23:54:57] INFO [users.views:115] User  accessed their profile.
[2024-11-21 23:54:57] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2169
[2024-11-21 23:54:58] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\views.py changed, reloading.
[2024-11-21 23:55:00] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:55:23] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\views.py changed, reloading.
[2024-11-21 23:55:25] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:55:25] INFO [users:12] User Anonymous accessed /users/profile/
[2024-11-21 23:55:25] INFO [users.views:115] User  accessed their profile.
[2024-11-21 23:55:25] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2169
[2024-11-21 23:55:30] INFO [users:12] User Anonymous accessed /users/profile/
[2024-11-21 23:55:30] INFO [users.views:115] User  accessed their profile.
[2024-11-21 23:55:30] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2169
[2024-11-21 23:55:38] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:55:40] INFO [users:12] User Anonymous accessed /users/profile/
[2024-11-21 23:55:40] INFO [users.views:115] User  accessed their profile.
[2024-11-21 23:55:40] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2169
[2024-11-21 23:55:43] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:55:43] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:56:37] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:56:40] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:56:40] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:57:14] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\ayu_site\settings.py changed, reloading.
[2024-11-21 23:57:15] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:57:16] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:57:16] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:58:21] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\forms.py changed, reloading.
[2024-11-21 23:58:23] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:58:38] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\app\ayu_site\users\views.py changed, reloading.
[2024-11-21 23:58:40] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-21 23:58:40] INFO [users:12] User Anonymous accessed /users/profile/
[2024-11-21 23:58:40] INFO [users.views:128] User  accessed their profile.
[2024-11-21 23:58:40] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2169
[2024-11-21 23:58:47] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 23:58:47] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-21 23:58:49] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:58:49] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4334
[2024-11-21 23:58:51] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 23:58:51] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-21 23:58:52] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:58:52] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-21 23:58:59] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-21 23:58:59] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4334
[2024-11-21 23:59:02] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 23:59:02] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-21 23:59:15] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-21 23:59:16] INFO [users.views:52] User admin logged in.
[2024-11-21 23:59:16] INFO [django.server:213] "POST /users/login/ HTTP/1.1" 302 0
[2024-11-21 23:59:16] INFO [users:12] User admin accessed /
[2024-11-21 23:59:16] INFO [django.server:213] "GET / HTTP/1.1" 200 3270
[2024-11-21 23:59:21] INFO [users:12] User admin accessed /users/profile/
[2024-11-21 23:59:21] INFO [users.views:128] User admin accessed their profile.
[2024-11-21 23:59:21] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 2953
[2024-11-21 23:59:24] INFO [users:12] User admin accessed /users/auth/logout/
[2024-11-21 23:59:24] INFO [django.server:213] "POST /users/auth/logout/ HTTP/1.1" 302 0
[2024-11-21 23:59:24] INFO [users:12] User Anonymous accessed /
[2024-11-21 23:59:24] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-24 15:19:24] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:22:25] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:23:11] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:24:38] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\settings.py changed, reloading.
[2024-11-24 15:24:38] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:24:42] INFO [users:12] User Anonymous accessed /
[2024-11-24 15:24:42] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\todo\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'home' not found. 'home' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\todo\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'home' not found. 'home' is not a valid view function or pattern name.
[2024-11-24 15:24:42] ERROR [django.server:213] "GET / HTTP/1.1" 500 201720
[2024-11-24 15:24:42] INFO [users:12] User Anonymous accessed /favicon.ico
[2024-11-24 15:24:42] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\todo\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'home' not found. 'home' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\todo\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'home' not found. 'home' is not a valid view function or pattern name.
[2024-11-24 15:24:42] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 201700
[2024-11-24 15:25:37] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\urls.py changed, reloading.
[2024-11-24 15:25:37] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:26:12] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:26:15] INFO [users:12] User Anonymous accessed /
[2024-11-24 15:26:15] INFO [django.server:213] "GET / HTTP/1.1" 200 2341
[2024-11-24 15:26:15] INFO [django.server:213] "GET /static/images/favicon.png HTTP/1.1" 200 470
[2024-11-24 15:50:29] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\urls.py changed, reloading.
[2024-11-24 15:50:29] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:52:29] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\Users\apps.py changed, reloading.
[2024-11-24 15:52:29] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:53:57] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:54:36] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 15:59:27] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:03:36] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:04:11] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:04:15] INFO [users:12] User Anonymous accessed /
[2024-11-24 16:04:15] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-24 16:05:24] INFO [users:12] User Anonymous accessed /
[2024-11-24 16:05:24] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-24 16:05:26] INFO [users:12] User Anonymous accessed /
[2024-11-24 16:05:26] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-24 16:05:33] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-24 16:05:33] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4334
[2024-11-24 16:05:34] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-24 16:05:34] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-24 16:05:36] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-24 16:05:36] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4334
[2024-11-24 16:05:37] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-24 16:05:37] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-24 16:13:24] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\settings.py changed, reloading.
[2024-11-24 16:13:24] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:31:43] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:41:32] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 16:43:08] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 17:11:51] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-24 19:28:28] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\settings.py changed, reloading.
[2024-11-24 19:28:28] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 12:00:19] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 12:00:23] INFO [users:12] User Anonymous accessed /
[2024-11-25 12:00:23] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-25 12:00:23] INFO [django.server:213] "GET /static/images/favicon.png HTTP/1.1" 200 470
[2024-11-25 12:11:05] INFO [users:12] User Anonymous accessed /
[2024-11-25 12:11:05] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-25 12:11:05] INFO [django.server:213] "GET /static/images/favicon.png HTTP/1.1" 304 0
[2024-11-25 13:35:39] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 13:35:42] INFO [users:12] User Anonymous accessed /
[2024-11-25 13:35:42] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-25 13:35:51] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-25 13:35:51] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-25 13:35:53] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-25 13:35:54] WARNING [users.views:62] Failed login attempt with username: root
[2024-11-25 13:35:54] INFO [django.server:213] "POST /users/login/ HTTP/1.1" 200 4151
[2024-11-25 13:36:00] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-25 13:36:00] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 4334
[2024-11-25 13:36:36] INFO [users:12] User Anonymous accessed /users/register/
[2024-11-25 13:36:37] INFO [users.views:34] New user registered: AyuArts
[2024-11-25 13:36:37] INFO [django.server:213] "POST /users/register/ HTTP/1.1" 302 0
[2024-11-25 13:36:37] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:36:37] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:36:37] ERROR [django.server:213] "GET / HTTP/1.1" 500 184840
[2024-11-25 13:36:37] INFO [users:12] User AyuArts accessed /favicon.ico
[2024-11-25 13:36:37] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:36:37] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 301032
[2024-11-25 13:37:07] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:37:07] INFO [django.server:213] "GET / HTTP/1.1" 200 3273
[2024-11-25 13:39:52] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:39:53] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:39:53] ERROR [django.server:213] "GET / HTTP/1.1" 500 184840
[2024-11-25 13:39:53] INFO [users:12] User AyuArts accessed /favicon.ico
[2024-11-25 13:39:53] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:39:53] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 301032
[2024-11-25 13:40:10] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:40:10] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.
[2024-11-25 13:40:10] ERROR [django.server:213] "GET / HTTP/1.1" 500 184965
[2024-11-25 13:40:10] INFO [users:12] User AyuArts accessed /favicon.ico
[2024-11-25 13:40:10] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.
[2024-11-25 13:40:10] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 301232
[2024-11-25 13:40:11] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:40:11] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.
[2024-11-25 13:40:11] ERROR [django.server:213] "GET / HTTP/1.1" 500 184965
[2024-11-25 13:40:11] INFO [users:12] User AyuArts accessed /favicon.ico
[2024-11-25 13:40:12] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'task/tasks' not found. 'task/tasks' is not a valid view function or pattern name.
[2024-11-25 13:40:12] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 301232
[2024-11-25 13:47:56] INFO [users:12] User Anonymous accessed /admin/users/user/
[2024-11-25 13:47:56] INFO [django.server:213] "GET /admin/users/user/ HTTP/1.1" 302 0
[2024-11-25 13:47:56] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-25 13:47:56] INFO [django.server:213] "GET /admin/login/?next=/admin/users/user/ HTTP/1.1" 200 4182
[2024-11-25 13:47:56] INFO [users:12] User Anonymous accessed /favicon.ico
[2024-11-25 13:47:56] WARNING [django.request:248] Not Found: /favicon.ico
[2024-11-25 13:47:56] WARNING [django.server:213] "GET /favicon.ico HTTP/1.1" 404 2291
[2024-11-25 13:50:25] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:50:25] ERROR [django.request:248] Internal Server Error: /
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:50:25] ERROR [django.server:213] "GET / HTTP/1.1" 500 184840
[2024-11-25 13:50:25] INFO [users:12] User AyuArts accessed /favicon.ico
[2024-11-25 13:50:25] ERROR [django.request:248] Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 25, in __call__
    return render(request, 'errors/404.html', status=404)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\home_work_20\users\middleware.py", line 28, in __call__
    return render(request, 'errors/500.html', status=500)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 327, in render
    return nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\loader_tags.py", line 210, in render
    return template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 173, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\template\defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'tasks' not found. 'tasks' is not a valid view function or pattern name.
[2024-11-25 13:50:25] ERROR [django.server:213] "GET /favicon.ico HTTP/1.1" 500 301032
[2024-11-25 13:52:59] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\urls.py changed, reloading.
[2024-11-25 13:53:00] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 13:53:02] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:53:02] INFO [django.server:213] "GET / HTTP/1.1" 200 3365
[2024-11-25 13:53:06] INFO [users:12] User AyuArts accessed /tasks/tasks/
[2024-11-25 13:53:06] INFO [django.server:213] "GET /tasks/tasks/ HTTP/1.1" 200 2992
[2024-11-25 13:53:10] INFO [users:12] User AyuArts accessed /users/auth/logout/
[2024-11-25 13:53:10] INFO [django.server:213] "POST /users/auth/logout/ HTTP/1.1" 302 0
[2024-11-25 13:53:10] INFO [users:12] User Anonymous accessed /
[2024-11-25 13:53:10] INFO [django.server:213] "GET / HTTP/1.1" 200 2345
[2024-11-25 13:53:14] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-25 13:53:14] INFO [django.server:213] "GET /users/login/ HTTP/1.1" 200 3829
[2024-11-25 13:53:30] INFO [users:12] User Anonymous accessed /users/login/
[2024-11-25 13:53:31] INFO [users.views:52] User AyuArts logged in.
[2024-11-25 13:53:31] INFO [django.server:213] "POST /users/login/ HTTP/1.1" 302 0
[2024-11-25 13:53:31] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:53:31] INFO [django.server:213] "GET / HTTP/1.1" 200 3365
[2024-11-25 13:53:36] INFO [users:12] User AyuArts accessed /tasks/tasks/
[2024-11-25 13:53:36] INFO [django.server:213] "GET /tasks/tasks/ HTTP/1.1" 200 2992
[2024-11-25 13:53:40] INFO [users:12] User AyuArts accessed /users/profile/
[2024-11-25 13:53:40] INFO [users.views:128] User AyuArts accessed their profile.
[2024-11-25 13:53:40] INFO [django.server:213] "GET /users/profile/ HTTP/1.1" 200 3052
[2024-11-25 13:56:18] INFO [users:12] User AyuArts accessed /tasks/tasks/
[2024-11-25 13:56:18] INFO [django.server:213] "GET /tasks/tasks/ HTTP/1.1" 200 2992
[2024-11-25 13:56:39] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 13:56:39] INFO [users:12] User AyuArts accessed /tasks/tasks/
[2024-11-25 13:56:39] WARNING [django.request:248] Not Found: /tasks/tasks/
[2024-11-25 13:56:39] WARNING [django.server:213] "GET /tasks/tasks/ HTTP/1.1" 404 3142
[2024-11-25 13:56:47] INFO [users:12] User AyuArts accessed /tasks/
[2024-11-25 13:56:47] INFO [django.server:213] "GET /tasks/ HTTP/1.1" 200 2986
[2024-11-25 13:56:50] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:56:50] INFO [django.server:213] "GET / HTTP/1.1" 200 3359
[2024-11-25 13:56:52] INFO [users:12] User AyuArts accessed /tasks/
[2024-11-25 13:56:52] INFO [django.server:213] "GET /tasks/ HTTP/1.1" 200 2986
[2024-11-25 13:57:05] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\home_work_20\urls.py changed, reloading.
[2024-11-25 13:57:07] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 13:57:08] INFO [users:12] User AyuArts accessed /tasks/
[2024-11-25 13:57:08] WARNING [django.request:248] Not Found: /tasks/
[2024-11-25 13:57:08] WARNING [django.server:213] "GET /tasks/ HTTP/1.1" 404 3148
[2024-11-25 13:57:10] INFO [users:12] User AyuArts accessed /
[2024-11-25 13:57:10] INFO [django.server:213] "GET / HTTP/1.1" 200 3365
[2024-11-25 13:57:12] INFO [users:12] User AyuArts accessed /users/tasks/
[2024-11-25 13:57:12] INFO [django.server:213] "GET /users/tasks/ HTTP/1.1" 200 2992
[2024-11-25 14:06:38] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\forms.py changed, reloading.
[2024-11-25 14:06:39] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:07:41] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\validators.py changed, reloading.
[2024-11-25 14:07:43] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:08:11] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\validators.py changed, reloading.
[2024-11-25 14:08:13] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:08:36] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\validators.py changed, reloading.
[2024-11-25 14:08:37] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:08:43] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\validators.py changed, reloading.
[2024-11-25 14:08:44] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:10:06] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\forms.py changed, reloading.
[2024-11-25 14:10:07] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:17:21] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\validators.py changed, reloading.
[2024-11-25 14:17:22] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:22:37] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 14:29:59] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\users\admin.py changed, reloading.
[2024-11-25 14:30:00] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 16:29:56] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\todo\admin.py changed, reloading.
[2024-11-25 16:30:03] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 16:40:45] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-25 16:40:50] INFO [users:12] User AyuArts accessed /
[2024-11-25 16:40:50] INFO [django.server:213] "GET / HTTP/1.1" 200 3365
[2024-11-25 16:41:54] INFO [users:12] User Anonymous accessed /admin/
[2024-11-25 16:41:54] INFO [django.server:213] "GET /admin/ HTTP/1.1" 302 0
[2024-11-25 16:41:54] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-25 16:41:54] INFO [django.server:213] "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4160
[2024-11-25 16:41:54] INFO [django.server:213] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/js/theme.js HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/css/login.css HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
[2024-11-25 16:41:55] INFO [django.server:213] "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
[2024-11-25 16:43:09] INFO [users:12] User Anonymous accessed /admin/login/
[2024-11-25 16:43:10] INFO [django.server:213] "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2024-11-25 16:43:10] INFO [users:12] User root accessed /admin/
[2024-11-25 16:43:10] INFO [django.server:213] "GET /admin/ HTTP/1.1" 200 7788
[2024-11-25 16:43:10] INFO [django.server:213] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[2024-11-25 16:43:10] INFO [django.server:213] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[2024-11-25 16:43:10] INFO [django.server:213] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [users:12] User root accessed /admin/django_db_logger/statuslog/
[2024-11-25 16:43:16] INFO [django.server:213] "GET /admin/django_db_logger/statuslog/ HTTP/1.1" 200 16847
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/css/changelists.css HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/jquery.init.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/core.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/actions.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/urlify.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/prepopulate.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/js/filters.js HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 304 0
[2024-11-25 16:43:16] INFO [django.server:213] "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 304 0
[2024-11-25 16:43:24] INFO [users:12] User root accessed /admin/todo/task/
[2024-11-25 16:43:24] ERROR [django.request:248] Internal Server Error: /admin/todo/task/
Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: todo_task.user_id

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\contrib\admin\options.py", line 2140, in changelist_view
    "selection_note": _("0 of %(cnt)s selected") % {"cnt": len(cl.result_list)},
                                                           ^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Base_data\_system_files\Practice\app\my_library\.venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: todo_task.user_id
[2024-11-25 16:43:24] ERROR [django.server:213] "GET /admin/todo/task/ HTTP/1.1" 500 174189
[2024-11-25 16:46:03] INFO [users:12] User root accessed /admin/todo/task/
[2024-11-25 16:46:03] INFO [django.server:213] "GET /admin/todo/task/ HTTP/1.1" 200 8700
[2024-11-25 16:46:03] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:46:03] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:46:08] INFO [users:12] User root accessed /admin/todo/task/add/
[2024-11-25 16:46:08] INFO [django.server:213] "GET /admin/todo/task/add/ HTTP/1.1" 200 13119
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/css/forms.css HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/js/calendar.js HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/css/widgets.css HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/js/change_form.js HTTP/1.1" 304 0
[2024-11-25 16:46:08] INFO [django.server:213] "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 304 0
[2024-11-25 16:46:20] INFO [users:12] User root accessed /admin/users/user/
[2024-11-25 16:46:20] INFO [django.server:213] "GET /admin/users/user/ HTTP/1.1" 200 12632
[2024-11-25 16:46:20] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:46:20] INFO [django.server:213] "GET /static/admin/img/search.svg HTTP/1.1" 304 0
[2024-11-25 16:46:20] INFO [django.server:213] "GET /static/admin/img/icon-yes.svg HTTP/1.1" 304 0
[2024-11-25 16:46:20] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:46:20] INFO [django.server:213] "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 304 0
[2024-11-25 16:46:20] INFO [django.server:213] "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
[2024-11-25 16:46:25] INFO [users:12] User root accessed /admin/users/user/1/change/
[2024-11-25 16:46:25] INFO [django.server:213] "GET /admin/users/user/1/change/ HTTP/1.1" 200 18491
[2024-11-25 16:46:25] INFO [django.server:213] "GET /static/admin/js/SelectBox.js HTTP/1.1" 304 0
[2024-11-25 16:46:25] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:46:25] INFO [django.server:213] "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 304 0
[2024-11-25 16:46:25] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:46:25] INFO [django.server:213] "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 304 0
[2024-11-25 16:46:25] INFO [django.server:213] "GET /static/admin/img/selector-icons.svg HTTP/1.1" 304 0
[2024-11-25 16:46:25] INFO [django.server:213] "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 304 0
[2024-11-25 16:47:02] INFO [users:12] User root accessed /admin/todo/task/
[2024-11-25 16:47:02] INFO [django.server:213] "GET /admin/todo/task/ HTTP/1.1" 200 8700
[2024-11-25 16:47:02] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:47:02] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:47:23] INFO [users:12] User root accessed /admin/todo/task/
[2024-11-25 16:47:23] INFO [django.server:213] "GET /admin/todo/task/ HTTP/1.1" 200 8700
[2024-11-25 16:47:23] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:47:23] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-25 16:47:26] INFO [users:12] User root accessed /admin/users/user/
[2024-11-25 16:47:26] INFO [django.server:213] "GET /admin/users/user/ HTTP/1.1" 200 12632
[2024-11-25 16:47:26] INFO [users:12] User root accessed /admin/jsi18n/
[2024-11-25 16:47:26] INFO [django.server:213] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2024-11-26 15:00:14] INFO [django.utils.autoreload:266] C:\Base_data\_system_files\Practice\home_work_20\todo\urls.py changed, reloading.
[2024-11-26 15:00:18] INFO [django.utils.autoreload:668] Watching for file changes with StatReloader
[2024-11-26 15:00:33] INFO [users:12] User AyuArts accessed /users/register/
[2024-11-26 15:00:34] INFO [django.server:213] "GET /users/register/ HTTP/1.1" 200 5179
